### PR TITLE
Feature/webhook encryption - Test

### PR DESCRIPTION
**Pull request by ParamonPlay2205**
This seemly works fine on my device used for testing.

### CODE REVIEW

File: common/build.gradle [hunk 1]
```diff
@@ -22,4 +22,7 @@ dependencies {
     kapt "com.google.dagger:dagger-compiler:${daggerVersion}"
 
     implementation "com.squareup.retrofit2:retrofit:$retrofit2Version"
+
+    implementation 'com.goterl.lazycode:lazysodium-android:4.1.0@aar'
```
Comment (JBassett) [inline common/build.gradle:26]: Looks like `4.2.0` is the latest?

**Review by JBassett — COMMENTED** [review]
Nice start!

**Comment by dshokouhi** [comment]
Is this for #97? We should probably check that every feature that makes a call to HA still works as expected. Not sure if we need to test against different network setups.

**Comment by dshokouhi** [comment]
I just noticed that Travis CI is added to the checks here.  Was this PR based off an older branch?

**Comment by balloob** [comment]
This PR seems to have gone stale. Closing it.

**Pull request closed without merging**